client/components/data/page/index.jsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies.
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
```

Comment:
Missed the dependency removal.

>(Also note to remove `bindActionCreators` dependency if you change this)

https://github.com/Automattic/wp-calypso/pull/4090#discussion_r57085958